src/index.test.ts

```diff
@@ -1180,14 +1180,48 @@ test('dedupe=true', () => {
   expect(json.a);
 
   // This has already been seen and should be deduped
-  expect(json.b).toBeNull();
+  expect(json.b).toEqual('[Pruned *]');
 
   expect(json).toMatchInlineSnapshot(`
     Object {
       "a": Object {
         "children": Array [],
       },
-      "b": null,
+      "b": "[Pruned *]",
+    }
+  `);
+
+  expect(instance.deserialize(output)).toEqual(input);
+});
+
+test.only('dedupe=true, circular', () => {
+  const instance = new SuperJSON({
+    dedupe: true,
+  });
+
+  type Node = {
+    children: Node[];
+  };
+  const root: Node = {
+    children: [],
+  };
+  root.children.push(root);
+  const input = {
+    a: root,
+    b: root,
+  };
+  const output = instance.serialize(input);
+
+  const json = output.json as any;
+
+  expect(json).toMatchInlineSnapshot(`
+    Object {
+      "a": Object {
+        "children": Array [
+          "[Circular *]",
+        ],
+      },
+      "b": "[Pruned *]",
```

Comment:
should this say `[Pruned <path-where-the-node-is>]` or something?

Reply:
Since we're adding this for the output to be more human-readable, I think we should take a string that's searchable + self-explanatory. I don't think `Pruned` cuts it. What do you think about something like`[SuperJSON: Equal to <path.where.node>]`?